src/components/FilterBar/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useEffect } from 'react'
+import Stack from '@mui/material/Stack'
+import MultiSelectCheckmarks from '../FilterControls/MultiSelect'
+
+import { Button } from '@mui/material'
+import {
+  MediaFilterActionTypes,
+  MediaFilterState,
+} from '../../types/interfaces/MediaFiltersReducer'
+import MediaTitleSearchInput from '../FilterControls/MediaTitleSearchInput'
+import MediaTypeRadioGroup from '../FilterControls/MediaTypeRadioGroup'
+
+const genreOptions = ['action', 'adventure', 'comedy']
```

Comment:
These probably ought to live alongside the respective components they're used for.